dbicdh/PostgreSQL/upgrade/97-98/001-auto.sql

```diff
@@ -0,0 +1,21 @@
+-- Convert schema '/home/adamw/local/openQA/script/../dbicdh/_source/deploy/97/001-auto.yml' to '/home/adamw/local/openQA/script/../dbicdh/_source/deploy/98/001-auto.yml':;
+
+;
+BEGIN;
+
+;
+DROP INDEX idx_value_settings;
+
+;
+DROP INDEX idx_job_id_value_settings;
```

Comment:
These two indexes do see a lot of use on our production instances, we should be very sure before limiting their scope:
```
 relid | indexrelid | schemaname |   relname    |       indexrelname        | idx_scan | idx_tup_read | idx_tup_fetch
-------+------------+------------+--------------+---------------------------+----------+--------------+---------------
 16513 |      17780 | public     | job_settings | idx_value_settings        |    38833 |      6342596 |         57192
 16513 |    2342927 | public     | job_settings | idx_job_id_value_settings | 42807954 |   1528570202 |    1036367663
```